### PR TITLE
GM Longitudinal: Display personality in instrument cluster

### PR DESCRIPTION
**Pull request by sunnyhaibin**
Displays Longitudinal Personality with 1st, 2nd, and 3rd bar.

**Requires**
- https://github.com/commaai/openpilot/pull/31760

### CODE REVIEW

File: selfdrive/car/gm/gmcan.py [hunk 1]
```diff
@@ -102,17 +102,17 @@ def create_friction_brake_command(packer, bus, apply_brake, idx, enabled, near_s
...
-def create_acc_dashboard_command(packer, bus, enabled, target_speed_kph, lead_car_in_sight, fcw):
+def create_acc_dashboard_command(packer, bus, enabled, target_speed_kph, hud_control, fcw):
   target_speed = min(target_speed_kph, 255)
 
   values = {
     "ACCAlwaysOne": 1,
     "ACCResumeButton": 0,
     "ACCSpeedSetpoint": target_speed,
-    "ACCGapLevel": 3 * enabled,  # 3 "far", 0 "inactive"
+    "ACCGapLevel": (hud_control.leadDistanceBars + 1) * enabled,  # 3 "far", 0 "inactive"
```
Comment (sshane) [inline selfdrive/car/gm/gmcan.py:112]: 3+1 is 4?

Comment (sshane) [inline selfdrive/car/gm/gmcan.py:112]:
```suggestion
    "ACCGapLevel": hud_control.leadDistanceBars * enabled,  # 3 "far", 0 "inactive"
```

3+1 is 4?

Comment (sunnyhaibin) [reply selfdrive/car/gm/gmcan.py:112]: Ah, was under the impression that `hud_control.leadDistanceBars` would be 0 to 2. Will fix.

**Comment by sshane** [comment]
@Verylukyguy can you try this?

**Pull request merged**